### DISCUSSION TITLE
Trouble opening downloaded files in SEB

### DISCUSSION THREAD

**Discussion by HashL7** [SafeExamBrowser/seb-win-refactoring#517] [Q&A]
Hey all,
Iam just getting back to using SEB, so I'm not entirely sure if this is a Bug Report or should this be posted in the discussion. (Sorry in advance.)

I installed SEB v 3.4.1 for Windows, on a Windows 10 64bit (v1809) machine and encounter the following issues;
 
1. When SEB is opened and running, it allows me to download files (docx/pptx/xlsx) but doesn't allow me to open them. I see the download list when I click on the SEB's download icon (Top Right corner) but I'm unable to open them from the download list. 
Is this how its supposed to be or is it bug? Or when we click on an item in the download list, it opens?  -- the only possible work around I managed to find is Opening the file manually from the downloads directory; I have included **Windows Explorer.exe**, **Winword.exe**, **Powerpnt.exe** and **Excel.exe** as Permitted Processes (checking - Icon on Taskbar, AutoStart, Allow running on background). 

2. this where I encounter my second problem; MS Office Files (docx/ppts/xlsx) when downloaded from the internet and opened (the first time), opens in a protected view; we usually have to click on **Enable Editing** that appears on a yellow banner on the top area of the window. I noticed that Iam unable to open these downloaded files on SEB, because there is an error message that pops up saying: _" Word experienced an error in trying to open the file.
Try these suggestions
check the file permissions for the document or drive
make sure there is sufficient free memory and disk space
open the file with the text recovery converter...."_
I did read from another discussion that someone found a possible work around by changing the setting from the Trust Center in Word/PowerPoint/Excel.... but that would involve changing for the Trust Center setting on each PC separately, no? Is there any other way? 

Really appreciate any solutions / comments / feedback.

**Comment by dbuechel** [comment]
Thanks for using SEB. Regarding your questions:

1. At the moment, we do not have a mechanism which would allow to open a downloaded file directly from within SEB. The procedure would be as you already indicated, namely do define e.g. MS Word as permitted third-party application and then start Word and open the downloaded file from Word directly. There is a feature request regarding this topic (#283), but we don't yet have a timeline for it.
2. This sounds like a MS Office issue, you might find some insight in the event log of Windows itself. To be sure, you could enable the live application log of SEB (under tab "Security" in the configuration tool) and then observe in real time when opening Word within SEB whether SEB is suppressing anything that would prevent Word from working correctly.

**Comment by wimglasseesjibe** [comment]
My 2 cents:

- Protected mode kicks in as expected when running SEB in 'Disable explorer shell' mode, 'Create new desktop' mode doesn't work. Would need a better understanding of what new desktop mode actually does to troubleshoot this behaviour.
- The 'easy' workaround is that you can open the properties dialog for this file, either from explorer if allowed or from within the microsoft word browse dialog, and 'unblock' it.
- Another option I tried and tested is through the powershell command Unblock-File.
e.g. 'Unblock-File ~\Downloads\*.*'.
This works and requires only 1 click if you add it as an application, but this obviously also has its downsides.

So, no clean solution yet...
